### ISSUE TITLE
Missing Symbol.toStringTag

### ISSUE THREAD

**Issue by jimmywarting** [WebReflection/url-search-params#38]

**Comment by WebReflection** [comment]
Pretty bad way to file an issue:

  * no content
  * no description
  * no test case
  * not a link

The result? Not an issue.

**Comment by jimmywarting** [comment]
I want to have the `Symbol.toStringTag` in the prototype So that I can identify what they pass to my function is a URLSearchParams

```js
new URLSearchParams()[Symbol.toStringTag] === 'URLSearchParams' // true
```

your lib returns `undefined` but the browser returns a string: `URLSearchParams`

**Comment by jimmywarting** [comment]
Are you saying a Symbol polyfill needs to fix the missing toStringTag?

**Comment by WebReflection** [comment]
this project is a polyfill for `URLSearchParams`

if the browser doesn't support `Symbol` or doesn't know `Symbol.toStringTag` this polyfill cannot fix that.

A `Symbol` polyfill should patch also all the constructors that don't use `Symbol` or its variants yet, like it's in the `Symbol.iterator` case and es6-shim https://github.com/paulmillr/es6-shim/blob/8583cbe2186ec12bfbf4218ccbce7fc73f96de87/es6-shim.js#L1264

TL;DR I won't patch `Symbol` in here and you can always use `obj.constructor.name` instead of `Symbol.toStringTag` for most common cases.

Browsers that support `Symbol.toStringTag` already support `URLSearchParams` and every other engine (beside IE8) supports `obj.constructor.name`

**Comment by jimmywarting** [comment]
what about node?

**Comment by WebReflection** [comment]
That assignment should eventually be:
```js
Object.defineProperty(URLSearchParams.prototype, Symbol.toStringTag, {configurable: true, value: 'URLSearchParams'});
```

PR welcome.
